governance/multisig_wh_message_builder/src/commander.ts

```diff
@@ -1,36 +1,39 @@
-import { ixFromRust, setDefaultWasm } from "@certusone/wormhole-sdk";
```

Comment:
Should this file keep the name index.ts

Reply:
I was planning to add two method for now in index.ts, createMultisigTx and executeMultisigTx [not final names]. and export them.

Anyone consuming them would do so using 
`import {createMultisigTx, executeMultisigTx} from "@pythnet/package_name";`

and not something like
`import {createMultisigTx, executeMultisigTx} from "@pythnet/package_name/file_name";`

we can leave commander as it is for no one will be using that file.
Does this make sense?

Reply:
I think `cli.ts` would be a good name for this file